elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/ElideAutoConfiguration.java

```diff
@@ -164,20 +170,47 @@ public Elide initializeElide(EntityDictionary dictionary,
         return new Elide(builder.build());
     }
 
+    /**
+     * A Set containing Types to be excluded from EntityDictionary's EntityBinding.
+     * @param settings Elide configuration settings.
+     * @return Set of Types.
+     */
+    @Bean
+    @ConditionalOnMissingBean
+    public Set<Type> getEntitiesToExclude(ElideConfigProperties settings) {
+        Set<Type> entitiesToExclude = new HashSet();
+
+        AsyncProperties asyncProperties = settings.getAsync();
+
+        if (asyncProperties == null || !asyncProperties.isEnabled()) {
+            entitiesToExclude.add(new ClassType(AsyncQuery.class));
+        }
+
+        boolean exportEnabled = isExportEnabled(asyncProperties);
+
+        if (!exportEnabled) {
+            entitiesToExclude.add(new ClassType(TableExport.class));
+        }
+
+        return entitiesToExclude;
+    }
+
```

Comment:
Should we have this code directly in `buildDictionary` bean

Reply:
This allows users to add their own models to the list.